server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -2905,4 +2905,21 @@ public static ResourceServer createResourceServer(ClientModel client, KeycloakSe
 
         return toModel(representation, authorization);
     }
+ 
+    public static void validateRealmRepresentation(RealmRepresentation rep, RealmModel realm, KeycloakSession session) throws ModelException {
+        int secondsPerYear = 31536000; // valid value condition the same as admin UI's partials/realm-tokens.html
```

Comment:
I am not 100% sure if 1 year is sufficient for all the use-cases? For example is it possible that someone uses "offline session" timeouts longer than 1 year? Should we rather increase to something like 10 years or Integer.MAX_VALUE?

What do others thing about this?

I agree that it will be good to have admin console validations in sync with the REST validations on server side. So whatever we agree on, we should make both places in sync.

Reply:
I am not sure which upper bound is really appropriate for every use case. Therefore, it might be preferable to set to upper bound Floor[Integer.MAX_VALUE (2,147,483,647) / seconds per year (31,536,000)] = 68 days.

WDYT?

Reply:
This works for me